src/main/webapp/app/shared/user-settings/vcs-access-tokens-settings/vcs-access-tokens-settings.html

```diff
@@ -0,0 +1,82 @@
+<h1 jhiTranslate="artemisApp.userSettings.vcsAccessTokenSettings">
+    <!--VCS access token Settings-->
+</h1>
+@if (currentUser) {
+<div class="list-group d-block">
+    <div class="list-group-item">
+        <dt>
+            <span jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.infoText"> </span>
+        </dt>
+    </div>
+    <div class="list-group-item">
+        <dt>
+            @if (this.currentUser?.vcsAccessToken) {
+            <table class="table table-striped flex">
+                <thead class="thead-dark">
+                    <tr>
+                        <th class="col-2">
+                            <span jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.vcsAccessToken"> </span>
+                        </th>
+                        <th class="col-1"></th>
+                        <th class="col-4">
+                            <span jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.expiryDate"> </span>
+                        </th>
+                        <th class="col-4">
+                            <span jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.actions"> </span>
+                        </th>
+                    </tr>
+                </thead>
+                <tbody class="">
+                    <tr>
+                        <td>***************</td>
+                        <td>
+                            <button
+                                [cdkCopyToClipboard]="currentUser!.vcsAccessToken || ''"
+                                (cdkCopyToClipboardCopied)="onCopyFinished($event)"
+                                [class.btn-success]="wasCopied"
+                                class="btn btn-secondary btn-sm me-2"
+                                type="button"
+                            >
+                                <fa-icon [icon]="faCopy"></fa-icon>
+                            </button>
+                        </td>
+                        <td>{{ this.currentUser?.vcsAccessTokenExpiryDate | artemisDate }}</td>
```

Comment:
```suggestion
                        <td jhiTranslate="this.currentUser?.vcsAccessTokenExpiryDate"></td>
```
Does this also work?

Reply:
Unfortunately not, as that's not a translate pipe. Do we have something similar to jhiTranslate for dates? I didn't find anything

Reply:
I just saw the pipe and immediately thought that it is for translating. My bad